modal/_utils/blob_utils.py

```diff
@@ -317,11 +324,16 @@ async def _download_from_url(download_url: str) -> bytes:
 
 
 async def blob_download(blob_id: str, stub: ModalClientModal) -> bytes:
-    # convenience function reading all of the downloaded file into memory
+    """Convenience function reading all of the downloaded file into memory."""
+    logger.debug(f"Downloading large blob {blob_id}")
+    t0 = time.time()
     req = api_pb2.BlobGetRequest(blob_id=blob_id)
     resp = await retry_transient_errors(stub.BlobGet, req)
-
-    return await _download_from_url(resp.download_url)
+    data = await _download_from_url(resp.download_url)
+    size_mib = len(data) / 1024 / 1024
+    throughput_mib_s = (len(data) / 1024 / 1024) / (time.time() - t0)
```

Comment:
Nit
```suggestion
    throughput_mib_s = size_mib / (time.time() - t0)

```

Reply:
Turns out you can get a `0` out of that duration calculation in tests too 😖